src/components/OptionsList/BaseOptionsList.tsx

```diff
@@ -1,89 +1,72 @@
-import PropTypes from 'prop-types';
+import {isEmpty, isEqual} from 'lodash';
```

Comment:
Are we able to use some native JS methods to replace lodash?

Reply:
I replaced `isEmpty` from `lodash` with `isEmptyString` from `StringUtils`.

However, I didn't find any replacement for the `isEqual` in our codebase, and `isEqual` from lodash is already used in a few typescript files.